core/src/main/java/org/jruby/util/cli/Options.java

```diff
@@ -186,6 +186,7 @@ public class Options {
     public static final Option<Boolean> USE_FIXNUM_CACHE = bool(MISCELLANEOUS, "fixnum.cache", true, "Use a cache of low-valued Fixnum objects.");
     public static final Option<Integer> FIXNUM_CACHE_RANGE = integer(MISCELLANEOUS, "fixnum.cache.size", 256, "Values to retrieve from Fixnum cache, in the range -X..(X-1).");
     public static final Option<Boolean> PACKED_ARRAYS = bool(MISCELLANEOUS, "packed.arrays", true, "Toggle whether to use \"packed\" arrays for small tuples.");
+    public static final Option<Integer> JAR_CACHE_UPDATE_CHECK_INTERVALL = integer(MISCELLANEOUS, "jarcache.updateCheck.intervall", 750, "The time (ms) between checks if a JAR file containing resources has been updated.");
```

Comment:
would have been nice to have some better naming e.g. `jar_cache.expiration.millis`
or at least consistency with casing (jarcache vs updateCheck) within one option and avoid typos (interval has one `l`)

Reply:
@kares oh yeah the typo has to change anyways.  Since we have fixnum.cache I think jar.cache.expiration and the description tells us the units.

Reply:
I've created another PR, just containing the requested name change (that I'm absolutely comfortable with).